scripts/cast-on-tenderly/index.js

```diff
@@ -0,0 +1,110 @@
+import 'dotenv/config';
+import axios from 'axios';
+import { Contract, ethers } from 'ethers';
+
+const NETWORK_ID = 5;
+const CHAINLOG_ADDRESS = '0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F';
+const CHIEF_HAT_SLOT = 12;
+const DEFAULT_TRANSACTION_PARAMETERS = { gasLimit: 1000000000 };
+
+// check env vars
+const REQUIRED_ENV_VARS = ['TENDERLY_USER', 'TENDERLY_PROJECT', 'TENDERLY_ACCESS_KEY'];
+if (REQUIRED_ENV_VARS.some(varName => !process.env[varName])) {
+    throw new Error(`Please provide all required env variables: ${REQUIRED_ENV_VARS.join(', ')}`);
+}
+
+// check process arguments
+const SPELL_ADDRESS = process.argv[2];
+if (!SPELL_ADDRESS) {
+    throw new Error('Please provide address of the spell, e.g.: `node index.js 0x...`');
+}
+
+const makeTenderlyApiRequest = async function (path) {
```

Comment:
Random question: why do you prefer this way for declaring a function instead of:

```js
async function makeTenderlyApiRequest(path) {}
```

or:

```js
const makeTenderlyApiRequest = async (path) => {}
```

Reply:
- To avoid unintentional use of hoisting (when function can be used before it is declared)
- `const` pattern also allows you to declare arrow functions the same way if you need to (because they are different from regular `function`s that arrow functions doesn't have `this` context)